ethereum-consortium-blockchain-network/azuredeploy.json

```diff
@@ -0,0 +1,737 @@
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "addressPrefix": "10.0.0.0/16",
+    "maxPeers": 25,
+    "mnSubnetNameArray": [
+      "[concat(uniqueString(concat(resourceGroup().id, concat(variables('namingInfix'), 'subnet')), 'mn0'))]",
+      "[concat(uniqueString(concat(resourceGroup().id, concat(variables('namingInfix'), 'subnet')), 'mn1'))]",
+      "[concat(uniqueString(concat(resourceGroup().id, concat(variables('namingInfix'), 'subnet')), 'mn2'))]",
+      "[concat(uniqueString(concat(resourceGroup().id, concat(variables('namingInfix'), 'subnet')), 'mn3'))]",
+      "[concat(uniqueString(concat(resourceGroup().id, concat(variables('namingInfix'), 'subnet')), 'mn4'))]"
+    ],
+    "mnSubnetPrefixArray": [
+      "10.0.0.0/24",
+      "10.0.1.0/24",
+      "10.0.2.0/24",
+      "10.0.3.0/24",
+      "10.0.4.0/24"],
+    "mnSubnetRefArray": [
+      "[concat(variables('vnetID'),'/subnets/', variables('mnSubnetNameArray')[0])]",
+      "[concat(variables('vnetID'),'/subnets/', variables('mnSubnetNameArray')[1])]",
+      "[concat(variables('vnetID'),'/subnets/', variables('mnSubnetNameArray')[2])]",
+      "[concat(variables('vnetID'),'/subnets/', variables('mnSubnetNameArray')[3])]",
+      "[concat(variables('vnetID'),'/subnets/', variables('mnSubnetNameArray')[4])]"],
+    "ubuntuImage": {
+      "publisher": "Canonical",
+      "offer": "UbuntuServer",
+      "sku": "14.04.4-LTS",
+      "version": "latest"
+    },
+    "apiVersion": "2015-06-15",
+    "apiVersionDeployments": "2016-07-01",
+    "bootMNNode": 0,
+    "mnNode": 1,
+    "txNode": 2,
+    "nodeKeyArray": [
+      "42fa516481d64692431fda09114912566a4e9b36bb5aae2ad52ef1bbbb212c59",
+      "40591f6282ace5b586e76abfba5641be515add41d39df7f1c86931ad046e9b60",
+      "66de5a329cbe4b45cf4e305c8920f99a683b70b56921166c5a3357c3b2da00e2",
+      "11b59a95b52ed643a6bfc9b738765fc53e030d7b9c40f524ab8e9b56a6d1953e",
+      "361fbc87dd053f50babc752c56bc1bd80f3a74338d970232800f8f52d82e5374"],
+    "nodeIDArray": [
+      "399fedb4bbedef3fc9077f6a8bd2b96112723c668e7b3072a802f15ca17622e799439470814f9b0bded21edd641eb4f727cd09836c09f38c6688e65b49edddcf",
+      "44ecec4c1c96cdd3ac77a04e9839278fc2db0979559244c38c2fe9494896e7eb574a2f040e401be09ad8ead5adee1915d2ec640016e253aa736a5958c10c476b",
+      "26661f1986f3b9c8bc28fd7aa8abcc5b6967421ccf3069f87d282d2aac040bf3c8b8e07784baf4e7b9b8b07edecf95b1da76e65f7db6612841011dffe154a142",
+      "9409170771c71385e67db347013db8b2820b3610fc00c9d777a66178a6554e7098ee16024148b6063b743c345d591f76962f84d5fd441aefaaaddc9a1d83fbda",
+      "f7460d34915d54aa716d330f64638e446fca843f1c84fbbc7f50b7f6fd67da375658c97b2829274893d21daa4ee5142773be74f57999985f3d83817e497295cd"],
+    "artifactsLocationURL": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/ethereum-consortium-blockchain-network"
+  },
+  "resources": [
+    {
+      "apiVersion": "[variables('apiVersion')]",
```

Comment:
Meed to have the apiVersion be literal api versions instead of variable for better schema validation